backend/internal/cmds/mgmt/serve/connect/cmd.go

```diff
@@ -584,3 +588,10 @@ func getAuthApiClientSecret() string {
 func getIsNeosyncCloud() bool {
 	return viper.GetBool("NEOSYNC_CLOUD")
 }
+
+func getAllowedWorkerApiKeys(isNeosyncCloud bool) []string {
+	if isNeosyncCloud {
```

Comment:
will only load api keys if in neosync cloud. why? well, that's it's only main usecase for right now.